client/src/components/Header/components/universal-nav.css

```diff
@@ -211,13 +216,25 @@ button.nav-link[aria-disabled='true'] {
   top: 0;
   overflow-y: auto;
   scrollbar-width: auto;
-  /* lang menu should always be full height */
+
+  /* Ensure lang menu scrolls if there is not enough height to display all of it (this ensures keyboard accessibility). 38px is the height of search form since it is inserted between header and main menu. */
+  height: min(
+    var(--lang-menu-height),
+    calc(100vh - 38px - var(--header-height))
```

Comment:
I am wondering, what is the reason to explicitly state `- 38xp`

we can go for something like 
```css
calc(100vh - calc(var(--header-height) *2))
```

Reply:
in the comment above - it says that the search form is 38px so that's where that comes from.